releng/maps/orbit_bundles.map

```diff
@@ -5,7 +5,7 @@ plugin@javax.xml,1.3.4=GET,http://download.eclipse.org/tools/orbit/downloads/dro
 
 plugin@javax.xml.rpc,1.1.0=GET,http://download.eclipse.org/tools/orbit/downloads/drops/R20130517111416/repository/plugins/javax.xml.rpc_1.1.0.v201209140446.jar,unpack=true,usetimestamp=true
 
-plugin@javax.xml.soap,1.2.0=GET,http://download.eclipse.org/tools/orbit/downloads/drops/R20130517111416/repository/plugins/javax.xml.soap_1.2.0.v201005080501.jar,unpack=true,usetimestamp=true
+plugin@javax.xml.soap,1.3.0=GET,http://download.eclipse.org/tools/orbit/downloads/drops/R20130517111416/repository/plugins/javax.xml.soap_1.3.0.v201105210645.jar,unpack=true,usetimestamp=true
```

Comment:
FYI, I can find no reference to this file.   I'm doubtful this is used in the build.  I can find these in the workspace (in target folders produced by the Tycho build):

![image](https://github.com/eclipse-birt/birt/assets/208716/f7b2d4ff-efe9-4810-977e-d979740b3dd6)

That's also what's in the target platform:

![image](https://github.com/eclipse-birt/birt/assets/208716/e86187f2-85df-4c1f-a73f-5b2eec459204)

So it's not clear if this change has any impact at all. I doubt it.

Reply:
Of course I think I will haven't an effect. The problem is that we need the version 1.3.0 in the lib-folder or the saaj.jar.
But the first try with saaj.jar crashed, the next try crashed at this moment to.

How can we get 1.3.0 or saaj.jar to the lib-folder?

Habe you an idea?

Reply:
I see it here:

![image](https://github.com/eclipse-birt/birt/assets/208716/0b350114-b988-4c85-af8c-17d79229357b)

It looks like it's already in the lib folder...

Reply:
It should be but in the latest build it isn't there for the OSGi-version.
(The standard-runtime has all libraries in the lib-folder and that is why it is running.)

Reply:
Have you and idea why the run was cancelled there is no reason listed at the log file.